etc/klavaro.profile

```diff
@@ -0,0 +1,55 @@
+# Firejail profile for klavaro
+# Description: Yet another touch typing tutor
+# This file is overwritten after every install/update
+# Persistent local customizations
+include klavaro.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.config/klavaro
+noblacklist ${HOME}/.local/share/klavaro
+
+include disable-common.inc
+include disable-devel.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/.local/share/klavaro
+mkdir ${HOME}/.config/klavaro
+whitelist ${HOME}/.local/share/klavaro
+whitelist ${HOME}/.config/klavaro
+include whitelist-common.inc
+
+apparmor
+caps.drop all
+machine-id
+netfilter
+no3d
+nodbus
+nodvd
+nogroups
+nonewprivs
+noroot
+notv
+nou2f
+novideo
+protocol unix
+seccomp
+shell none
+tracelog
+
+disable-mnt
+private-bin klavaro
```

Comment:
add tclsh
edit: more are missing: execvp: No such file or directory

Reply:
interesting, I don't have any problems or messages like this.
`firejail --profile=klavaro.profile --debug klavaro 2>&1 | grep execvp`
`execvp argument 0: klavaro`

Reply:
wow, you're correct
using your profile it works
but I was just testing no profile with `private-bin klavaro` and it wouldn't launch

weird

adding tclsh and bash get it going without a profile